src/main/java/seedu/recruittrackpro/logic/commands/FindCommand.java

```diff
@@ -22,7 +25,10 @@ public class FindCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Finds all persons whose names or attributes contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: " + PREFIX_NAME + " or " + PREFIX_TAG + "KEYWORD [MORE_KEYWORDS]...\n"
+            + "Parameters: "
+            + PREFIX_NAME + " or " + PREFIX_TAG + " or " + PREFIX_ADDRESS
+            + " or " + PREFIX_EMAIL + " or " + PREFIX_PHONE
+            + "KEYWORD [MORE_KEYWORDS]...\n"
             + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "alice bob charlie";
```

Comment:
I agree with Delon's comment, consider the following `MESSAGE_USAGE`:

```suggestion
public static final String MESSAGE_USAGE = COMMAND_WORD
            + ": Finds all persons whose names or attributes contain any of "
            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
            + "Parameters: "
            + PREFIX_NAME + " NAME [MORE_NAMES] or \n"
            + PREFIX_TAG + " TAG [MORE_TAGS] or \n"
            + PREFIX_ADDRESS + " ADDRESS [MORE_ADDRESSES] or \n"
            + PREFIX_EMAIL + " EMAIL [MORE_EMAILS] or \n"
            + PREFIX_PHONE + " PHONE [MORE_PHONES]\n"
            + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "alice bob charlie"
            + " " + PREFIX_EMAIL + "alice@gmail.com bob@yahoo.com charlie@hotmail.com";